experimental/visual-dataflow/src/DOTReformat.cpp

```diff
@@ -0,0 +1,69 @@
+#include <fstream>
+#include <string>
+#include <vector>
+
+void reformatDot(const std::string &inputFileName,
+                 const std::string &outputFileName) {
+  std::ifstream inputFile(inputFileName);
+  std::ofstream outputFile(outputFileName);
+
```

Comment:
You should probably check that these files were "opened" correctly (don't remember exactly the syntax, look up the documentation online)